src/index.js

```diff
@@ -91,33 +104,37 @@ if (fs.existsSync(fullPathToComponentDir)) {
 // Start by creating the directory that our component lives in.
 mkDirPromise(componentDir)
   .then(() => (
-    readFilePromiseRelative(templatePath)
+    generateTemplate(componentName, program.type, program.styling)
   ))
   .then(template => {
     logItemCompletion('Directory created.');
     return template;
   })
   .then(template => (
-    // Replace our placeholders with real data (so far, just the component name)
-    template.replace(/COMPONENT_NAME/g, componentName)
-  ))
-  .then(template => (
     // Format it using prettier, to ensure style consistency, and write to file.
-    writeFilePromise(filePath, prettify(template))
+    writeFilePromise(componentFilePath, prettify(template))
   ))
-  .then(template => {
+  .then(() => {
+    // Check whether we should make a CSS file too
+    if (hasCssFile(program.styling)) {
+      // Format it using prettier, to ensure style consistency, and write to file.
+      return writeFilePromise(cssFilePath, prettifyCss(cssTemplate))
```

Comment:
Nit: I think it'd be better to move the line that defines `cssFilePath` (line 68) to within this `if` statement.

Just because it isn't used most of the time, so it feels a little cleaner to not have this unused constant defined for every execution.

Reply:
I think it should be like what you said! But I thought maybe you'll like to put it beside other paths variables :) Sorry!